tests/Unit/PointwiseFunctions/AnalyticData/GrMhd/Test_SphericalTorus.cpp

```diff
@@ -0,0 +1,170 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#include "Framework/TestingFramework.hpp"
+
+#include <array>
+#include <random>
+
+#include "Framework/TestHelpers.hpp"
+#include "Helpers/DataStructures/MakeWithRandomValues.hpp"
+#include "Helpers/Domain/CoordinateMaps/TestMapHelpers.hpp"
+#include "PointwiseFunctions/AnalyticData/GrMhd/SphericalTorus.hpp"
+#include "Utilities/StdArrayHelpers.hpp"
+
+namespace {
+const std::array<double, 3> t2a(const tnsr::I<double, 3>& tens);
+const tnsr::I<double, 3> a2t(const std::array<double, 3>& arr);
+
+const std::array<double, 3> t2a(const tnsr::I<double, 3>& tens) {
+  std::array<double, 3> arr{};
+  for (size_t i = 0; i < 3; i++) {
+    gsl::at(arr,i) = tens.get(i);
+  }
+  return arr;
+}
+
+const tnsr::I<double, 3> a2t(const std::array<double, 3>& arr) {
```

Comment:
Remove `const` from return type.  No reason to restrict what the caller can do with the temporary.  Also on `t2a`.